src/crimson/osd/ops_executer.cc

```diff
@@ -478,12 +478,20 @@ OpsExecuter::execute_op(OSDOp& osd_op)
 {
   return (osd_op.op.flags & CEPH_OSD_OP_FLAG_FAILOK) ?
     do_execute_op(osd_op).handle_error_interruptible(
-      crimson::ct_error::eagain::pass_further{},
-      crimson::ct_error::einprogress::pass_further{},
-      osd_op_errorator::all_same_way([] {
-        return osd_op_errorator::now();
+      osd_op_errorator::all_same_way([&osd_op] (auto e, auto&& e_raw)
+      -> OpsExecuter::osd_op_errorator::future<> {
```

Comment:
Seems like it would be simpler to handle the CEPH_OSD_OP_FLAG_FAILOK within the all_same_way since both branches are identical up to that point?

Reply:
done.